packages/odyssey-react-mui/src/FieldError.tsx

```diff
@@ -14,24 +14,30 @@ import { memo } from "react";
 
 import { FormHelperText } from "@mui/material";
 import { ScreenReaderText } from "./ScreenReaderText";
+import { ErrorMessagesList } from "./ErrorMessagesList";
 import { useTranslation } from "react-i18next";
 
 import type { SeleniumProps } from "./SeleniumProps";
+import { Box } from "./Box";
 
 export type FieldErrorProps = {
   id?: string;
-  text: string;
+  message?: string;
+  messagesList?: string[];
 } & SeleniumProps;
 
-const FieldError = ({ id, testId, text }: FieldErrorProps) => {
+const FieldError = ({ id, message, messagesList, testId }: FieldErrorProps) => {
   const { t } = useTranslation();
 
   return (
     <FormHelperText data-se={testId} error id={id}>
       <ScreenReaderText>{`${t(
         "fielderror.screenreader.text"
       )}:`}</ScreenReaderText>
-      {text}
+      <Box>
```

Comment:
In the case that a single error message and a list are both passed in, they had to be wrapped in a `Box` to prevent them from being rendered side by side instead of stacked

Reply:
@trevoring-okta We should type the props so that `message` and `messageList` are mutually exclusive

Reply:
Something like:
```js
& {
  message: string,
  messagesList? never,
}
| {
  message?: never,
  messagesList: string[],
} & AllowedProps;```